pkgs/servers/nosql/influxdb2/default.nix

```diff
@@ -0,0 +1,125 @@
+{ buildGoModule
+, buildGoPackage
+, fetchFromGitHub
+, git
+, go-bindata
+, lib
+, llvmPackages
+, mkYarnPackage
+, pkg-config
+, rustPlatform
+, ... }:
+let
+  version = "2.0.2";
+  libflux_version = "0.95.0";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "influxdata";
+    repo = "influxdb";
+    sha256 = "1vv0f5zpxl7s6rsadhsj25zxk04m7a0jzghrbzynjv7xv7ml90r7";
+    # The build process requires `git rev-parse HEAD` to work, so we
+    # ask for submodules (to force the use of fetchGit) and tell it to
+    # leave .git in place. A shallow clone is fine though.
+    fetchSubmodules = true;
+    leaveDotGit = true;
+  };
+
+  ui = mkYarnPackage {
+    src = src;
+    packageJSON = "${src}/ui/package.json";
+    yarnLock = "${src}/ui/yarn.lock";
+    extraBuildInputs = [ git ];
```

Comment:
It would be great if we could remove this.

Reply:
Removed.